secret_harvest/github_manager.py

```diff
@@ -51,7 +51,7 @@ def search_code_repositories(self,
             for page in random_pages:
 
                 url = (f"https://api.github.com/search/repositories"
-                       f"?q={keyword}&page={page}")
+                       f"?q={keyword}")
```

Comment:
Fix bug where page number might be very high. Github only provides first 1000 results.